webapp/components/ContributionForm/ContributionForm.vue

```diff
@@ -10,13 +10,15 @@
           />
         </hc-teaser-image>
         <ds-input model="title" class="post-title" placeholder="Title" name="title" autofocus />
+        <small style="width:100%;position:relative;left:90%">{{ form.title.length }}/64</small>
         <no-ssr>
           <hc-editor
             :users="users"
             :hashtags="hashtags"
             :value="form.content"
             @input="updateEditorContent"
           />
+          <small style="width:100%;position:relative;left:90%">{{ form.contentLength }}/2000</small>
```

Comment:
I would like more to have this not a hard code max length and been a speaking and centralised changeable const …

Ah, you have defined it here: `this.formSchema.content.max`